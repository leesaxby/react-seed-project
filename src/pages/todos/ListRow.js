import React from 'react';
import List from './listRow/List';

export default function ListRow() {
    return (
        <List />
    );
}
