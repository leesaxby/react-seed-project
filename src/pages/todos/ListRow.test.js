import ListRow from './ListRow';

describe('A ListRow', () => {
	it('renders without throwing an error');
	it('renders a List within a Row');
});