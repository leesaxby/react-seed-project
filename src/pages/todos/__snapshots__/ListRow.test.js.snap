// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A ListRow renders component 1`] = `
<div
  className="row"
>
  <div
    className="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2"
  >
    <List />
  </div>
</div>
`;
